opencog/relex/Dockerfile

```diff
@@ -0,0 +1,98 @@
+#
+# Docker file that builds RelEx and starts the RelEx server.
+#
```

Comment:
Or we could keep things manual.  The only thing that is scarier to me than duplicated code is automatically-generated rpms and debs :-)  I mean, all hail rolling releases but... I want to minimize complexity. Things are already hard-enough to debug.